tests/validation/tests/v3_api/test_pipeline.py

```diff
@@ -0,0 +1,100 @@
+from .common import get_global_admin_client_and_cluster, create_kubeconfig, \
+    random_test_name, create_project_and_ns, get_project_client_for_token, \
+    random_str, wait_for_condition, get_cluster_client_for_token, \
+    create_connection, WebsocketLogParse, CATTLE_TEST_URL, USER_TOKEN
+import pytest
+import urllib
+from threading import Lock
+import time
+
+#RANCHER_CLUSTER NAME, CATTLE_TEST_URL, USER_TOKEN
```

Comment:
Can we remove this comment if its not necessary?

Reply:
removed